### ISSUE TITLE
Unbinding a key causes errors

### ISSUE THREAD

**Issue by Jan-PieterBaert** [qutebrowser/qutebrowser#4948]
**Version info (see `:version`)**:
```
qutebrowser v1.7.0
Git commit: c14f64619 (2019-07-30 12:30:10 +0200)
Backend: QtWebEngine (Chromium 73.0.3683.105)

CPython: 3.7.4
Qt: 5.13.0
PyQt: 5.13.0

sip: 4.19.18
colorama: no
pypeg2: 2.15
jinja2: 2.10.1
pygments: 2.4.2
yaml: 5.1.1
cssutils: no
attr: 19.1.0
PyQt5.QtWebEngineWidgets: yes
PyQt5.QtWebKitWidgets: yes
pdf.js: 2.2.228 (/usr/share/pdf.js/build/pdf.js)
sqlite: 3.29.0
QtNetwork SSL: OpenSSL 1.1.1c  28 May 2019

Style: QFusionStyle
Platform: Linux-5.2.5-arch1-1-ARCH-x86_64-with-arch, 64bit
Linux distribution: Arch Linux (arch)
Frozen: False
Imported from /usr/lib/python3.7/site-packages/qutebrowser
Using Python from /usr/bin/python3
Qt library executable path: /usr/lib/qt/libexec, data path: /usr/share/qt

Paths:
cache: /home/jan-pieter/.cache/qutebrowser
config: /home/jan-pieter/.config/qutebrowser
data: /home/jan-pieter/.local/share/qutebrowser
runtime: /run/user/1000/qutebrowser
system data: /usr/share/qutebrowser

Uptime: 1:31:30
```

**Does the bug happen if you start with `--temp-basedir`?**:
Yes

**Description**
When unbinding a key in qutebrowser, it adds a line like `config.bind('a', None)` to unbind the key a. But this causes the following issue when reading the config-file
```
ERROR: Errors occurred while reading config.py:
  Unhandled exception - AttributeError: 'NoneType' object has no attribute 'strip'
```

**How to reproduce**
unbind a key, write out the config file and read that config file again

**Comment by The-Compiler** [comment]
Reproduced via `:unbind H` (`a` isn't bound by default) and `:config-write-py`. Stack:

```
  File "/home/florian/proj/qutebrowser/git/qutebrowser/config/configfiles.py", line 629, in read_config_py
    exec(code, module.__dict__)
  File "/tmp/unbind/config/config.py", line 22, in <module>
    config.bind('H', None)
  File "/home/florian/proj/qutebrowser/git/qutebrowser/config/configfiles.py", line 447, in bind
    self._keyconfig.bind(seq, command, mode=mode)
  File "/home/florian/proj/qutebrowser/git/qutebrowser/config/config.py", line 205, in bind
    if not command.strip():
```

Note that this is a kind of exotic thing though - you'd normally only use `:config-write-py` if you want to switch from `autoconfig.yml` (which is used by `:bind` and `:set`) to a `config.py` file.

**Comment by Jan-PieterBaert** [comment]
I just tried `config.unbind(key)` and that seems to work, so a solution would be to let `:config-write-py` write `config.unbind(key)` instead of `config.bind(key, None)`

**Comment by The-Compiler** [comment]
Fixed via #4949
